packages/cli/src/commands/process/delete.ts

```diff
@@ -21,13 +24,15 @@ export default class ProcessDelete extends Command {
 
   async run(): Promise<string[]> {
     const {argv, flags} = this.parse(ProcessDelete)
+    const { account, mnemonic } = await this.getAccount(flags.account)
     if (!flags.confirm && !await cli.confirm('Are you sure?')) return []
```

Comment:
i really don't like this question because there is no default answer.
Let's either remove it because now there is an account to choose or add a default answer if possible

Reply:
there is a default if you passe the flag `--account` but yes you will need to type your password. I'm not sure I really understand the issue here.
Also, this account management will disappear in the next version of the CLI so I wouldn't spend too much time on that.

Reply:
I'm speaking about `cli.confirm('Are you sure?')`
it should be something like `cli.confirm('Are you sure? [Y/n]')` and by pressing enter key, confirm because it's the default